app/components/Button.js

```diff
@@ -0,0 +1,14 @@
+const Button = props => (
+  <button
+    type={props.type}
+    className={`btn btn--${props.color} btn--${props.size}`}
+    onClick={props.handleClick}
+    aria-label={props.ariaLabel}
+    aria-haspopup={props.ariaHasPopup}
+    aria-expanded={props.ariaExpanded}
+    aria-pressed={props.ariaPressed}
+    disabled={props.disabled}>
+      {props.label}
+  </button>)
+
+export default Button
```

Comment:
Following the pattern of the badge and alert components, wouldn't the structure be:
```
const Button = ({
      type,
      color,
      size,
      handleClick,
      label,
      ariaLabel
      ariaHasPopup,
      ariaExpanded,
      ariaPressed,
      disabled
    }) => (
      <button
        type={type}
        className={`btn btn--${color} btn--${size}`}
        onClick={handleClick}
        aria-label={ariaLabel}
        aria-haspopup={ariaHasPopup}
        aria-expanded={ariaExpanded}
        aria-pressed={ariaPressed}
        disabled={disabled}
      >
        {label}
      </button>
    );

Reply:
It's _kind_ of a preference thing - there's no hard rule about when you expand props and when you don't, but generally speaking when there are this many props I don't expand them, just for concision's sake.